homework 1/ReaderWriterLock/SharedReourceTests.cs

```diff
@@ -19,17 +20,28 @@ public void TestConcurrentReadWrite()
         // Реализовать проверку конкурентной записи и чтения, где в конце должны проверить что данные последнего потока записаны
         // Проверка должна быть многопоточной.
         // Потоков чтения должно быть ReadersThreads, потоков записи должно быть WritersThreads
-        
-        ClassicAssert.AreEqual($"Data {WritersThreads-1}", _sharedResource.Read());
+        var shared = new SharedResourceLock();
+        ConcurrentReadWrite(shared, WritersThreads, ReadersThreads);
     }
-    
+
     [Test]
     public void TestConcurrentReadWriteRwLock()
     {
         // Реализовать проверку конкурентной записи и чтения, где в конце должны проверить что данные последнего потока записаны
         // Проверка должна быть многопоточной
         // Потоков чтения должно быть ReadersThreads, потоков записи должно быть WritersThreads
-        
-        ClassicAssert.AreEqual($"Data {WritersThreads-1}", _sharedResource.Read());
+        var shared = new SharedResourceLock();
```

Comment:
наверное должно быть
```suggestion
        var shared = new SharedResourceRwLock();
```